websites/K/Kahoot/presence.ts

```diff
@@ -1,61 +1,84 @@
-			stString: "kahoot.stString",
-			ndString: "kahoot.ndString",
-			rdString: "kahoot.rdString",
-			topX: "kahoot.topX",
-			of: "kahoot.of",
-			questionNumber: "kahoot.questionNumber"
+			buttonJoinGame: "kahoot.buttonJoinGame", // Join Game: {0}
+			joiningGame: "kahoot.joiningGame", // Joining Game...
+			waiting: "kahoot.waiting", // Waiting to Start...
+			gameStarting: "kahoot.gameStarting", // Game Starting!
+			playing: "kahoot.playing", // Playing...
+			questionLoading: "kahoot.questionLoading", // Next Question Loading...
+			incorrectAnswer: "kahoot.incorrectAnswer", // Incorrect Answer
+			correctAnswer: "kahoot.correctAnswer", // Correct Answer
+			pollAnswer: "kahoot.pollAnswer", // + Answered Poll
+			resultsQuestion: "kahoot.resultsQuestion", // Looking at Results:
+			slideViewing: "kahoot.slideViewing", // Viewing Slide with Content...
+			gameOver: "kahoot.gameOver", // Game Over
+			gameCreate: "kahoot.gameCreate", // Creating a Game...
+			loadingPage: "kahoot.loadingPage", // Loading Page...
+			firstPlace: "kahoot.firstPlace", // First Place: {0}
+			points: "kahoot.points", // Points: {0}
+			questionsCorrect: "kahoot.questionsCorrect", // Questions Correct: {0}
+			slideShowing: "kahoot.slideShowing", // Showing Slide with Content
+			questionShowing: "kahoot.questionShowing", // Showing Question:
+			stString: "kahoot.stString", // 1st place
+			ndString: "kahoot.ndString", // 2nd place
+			rdString: "kahoot.rdString", // 3rd place
+			topX: "kahoot.topX", // Top {0}
+			onPodium: "kahoot.onPodium", // On the Podium
+			of: "kahoot.of", // {0} of {1}
+			questionNumber: "kahoot.questionNumber", // Question: {0}
+			feedback: "kahoot.feedback", // Providing feedback
+			waitingAnswer: "kahoot.waitingAnswer", // Waiting for results...
+			drumRoll: "kahoot.drumRoll", // Waiting for final ranking...
+			position: "kahoot.position", // Position: {0}
+			teamTalk: "kahoot.teamTalk", // Discussing with team
+			gameSummary: "kahoot.gameSummary", // Looking at game summary
+			login: "kahoot.login", // Logging in
+			createHome: "kahoot.createHome", // Viewing Create home Page
+			discover: "kahoot.discover", // Viewing Kahoot! discover Page
+			searchKahoots: "kahoot.searchKahoots", // Searching for Kahoots:
+			kahootDetails: "kahoot.kahootDetails", // Viewing Kahoot details:
+			kahootProfile: "kahoot.kahootProfile", // Viewing Kahoot profile:
+			myKahoots: "kahoot.myKahoots", // Viewing their Kahoots
+			userReports: "kahoot.userReports", // Viewing game reports
+			myCourses: "kahoot.myCourses", // Viewing their courses
+			editingCourse: "kahoot.editingCourse", // Editing a course:
+			viewingCourse: "kahoot.viewingCourse", // Viewing a course:
+			editingKahoot: "kahoot.editingKahoot", // Editing a Kahoot:
+			previewingKahoot: "kahoot.previewingKahoot", // Previewing a Kahoot
+			liveCourse: "kahoot.liveCourse", // Conducting a live course
+			liveCourseActivity: "kahoot.liveCourseActivity" // Activity #{0}: {1}
```

Comment:
Comments like such are not used anywhere else in the repository so should not be used here either. A future maintainer can reference the strings from PreMiD/Localization without much inconvenience.